openpype/lib/execute.py

```diff
@@ -74,18 +74,23 @@ def execute(args,
     return popen.returncode
 
 
-def run_subprocess(*args, **kwargs):
+def run_subprocess(*args, creationflags=None, **kwargs):
     """Convenience method for getting output errors for subprocess.
 
     Output logged when process finish.
 
     Entered arguments and keyword arguments are passed to subprocess Popen.
 
+    Set 'creationflags' to '0' (int) if auto-fix for creation of new window
+    should be ignored.
+
     Args:
-        *args: Variable length arument list passed to Popen.
+        *args: Variable length argument list passed to Popen.
+        creationflags (int): Creation flags for 'subprocess.Popen'.
```

Comment:
I was wondering why have a specific argument, instead of reading it from the kwargs?

Reply:
You're right. Changed.